### PR TITLE
test(em) add tests for externalTallies utils library

### PR DESCRIPTION
**Pull request by carolinemodic**

### CODE REVIEW

File: apps/election-manager/src/App.test.tsx [hunk 1]
```diff
@@ -279,7 +280,7 @@ test('tabulating CVRs', async () => {
 
   fireEvent.click(getByText('Tally'))
   fireEvent.click(getByText('Clear All Results…'))
-  fireEvent.click(getByText('Remove All Files'))
+  fireEvent.click(getByText('Remove All Data'))
```
Comment (benadida) [inline apps/election-manager/src/App.test.tsx:283]: was this test working before? How come?

File: apps/election-manager/src/App.test.tsx [hunk 2]
```diff
@@ -448,9 +630,9 @@ test('clearing all files after marking as official clears SEMS and CVR file', as
 
   fireEvent.click(getByText('Clear All Results…'))
   getByText(
-    'Do you want to remove the 1 uploaded CVR file and the external results file sems-results.csv?'
+    'Do you want to remove the 1 uploaded CVR file, the external results file sems-results.csv, and the manually entered data?'
   )
-  fireEvent.click(getByText('Remove All Files'))
+  fireEvent.click(getByText('Remove All Data'))
```
Comment (benadida) [inline apps/election-manager/src/App.test.tsx:635]: same question here.

File: apps/election-manager/src/components/ConfirmRemovingFileModal.tsx [hunk 3]
```diff
@@ -78,31 +88,39 @@ export const ConfirmRemovingFileModal: React.FC<Props> = ({
...
       break
     }
+    case ResultsFileType.Manual: {
+      fileTypeName = 'Manual Data'
+      mainContent = <p>Do you want to remove the manually entered data?</p>
+      break
+    }
     case ResultsFileType.All: {
-      fileTypeName = ''
+      fileTypeName = 'Data'
```
Comment (benadida) [inline apps/election-manager/src/components/ConfirmRemovingFileModal.tsx:107]: ahah, maybe this is where the change matches the tests?

File: apps/election-manager/src/components/ConfirmRemovingFileModal.tsx [hunk 4]
```diff
@@ -121,7 +139,7 @@ export const ConfirmRemovingFileModal: React.FC<Props> = ({
         <React.Fragment>
           <Button onPress={onClose}>Cancel</Button>
           <Button danger onPress={() => resetFiles(fileType)}>
-            Remove {!singleFileRemoval && 'All'} {fileTypeName} Files
+            Remove {!singleFileRemoval && 'All'} {fileTypeName}
```
Comment (benadida) [inline apps/election-manager/src/components/ConfirmRemovingFileModal.tsx:142]: ah yes, got it.

**Review by benadida — COMMENTED** [review]
@carolinemodic does this need to rebased? It looks like it's both the test and feature commits, but I think this PR is just for the tests?

**Comment by carolinemodic** [comment]
Ah yeah sorry, will rebase sent this out before some of the other things were merged.

**Review by beausmith — APPROVED** [review]
This all looks great!

One request: update the `coverageThreshold` percentages in `package.json` to include new coverage values. I assume they've increased!

**Pull request merged**